autoscale/src/main/java/com/sequenceiq/periscope/controller/validation/AlertValidator.java

```diff
@@ -127,6 +128,25 @@ public void validateClusterLimit(AdjustmentType adjustmentType, Integer configur
         }
     }
 
+    public void validateScheduleWithStopStartAndDisable(Cluster cluster, DistroXAutoscaleClusterRequest autoscaleClusterRequest) {
+        if (!autoscaleClusterRequest.getTimeAlertRequests().isEmpty() &&
+                (Boolean.TRUE.equals(autoscaleClusterRequest.getUseStopStartMechanism()) || Boolean.TRUE.equals(cluster.isStopStartScalingEnabled()))) {
+            if (Boolean.TRUE.equals(cluster.isStopStartScalingEnabled())) {
+                asClusterCommonService.setStopStartScalingState(cluster.getId(), Boolean.FALSE);
+            }
+            throw new BadRequestException(messagesService.getMessage(MessageCode.VALIDATION_TIME_STOP_START_UNSUPPORTED));
+        }
+    }
+
+    public void validateScheduleWithStopStartAndDisable(Cluster cluster, AutoscaleClusterState autoscaleState) {
+        if (Boolean.TRUE.equals(autoscaleState.getUseStopStartMechanism()) && !cluster.getTimeAlerts().isEmpty()) {
+            if (Boolean.TRUE.equals(cluster.isStopStartScalingEnabled())) {
+                asClusterCommonService.setStopStartScalingState(cluster.getId(), Boolean.FALSE);
```

Comment:
Same question about why we need to disable this, and not directly throw an error?

Reply:
Updated this to just throw the exception